classes/requests/helpers/class-nets-easy-checkout-helper.php

```diff
@@ -182,4 +185,75 @@ public static function get_phone_number( $order ) {
 		}
 		return $phone_number;
 	}
+
+
+	/**
+	 * Gets customer address.
+	 *
+	 * @return array
+	 */
+	protected static function get_em_customer_data() {
```

Comment:
* Instead of using `WC()->checkout()->get_value( 'billing_country' )` I think we should use `WC()->customer->get_billing_country()`. That information will probably be available earlier in the WC process.